packages/react/src/menu-item/MenuItem.tsx

```diff
@@ -0,0 +1,54 @@
+import * as RadixMenu from "@radix-ui/react-dropdown-menu";
+import { type ReactNode, forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+import { Flex } from "../flex";
+import { Text } from "../text";
+import * as styles from "./MenuItem.css";
+
+type MenuItemProps = BoxProps<
+  typeof RadixMenu.Item,
+  {
+    endDecorator?: ReactNode;
+    startDecorator?: ReactNode;
+  }
+>;
+
+export const MenuItem = forwardRef<HTMLDivElement, MenuItemProps>(
+  (
+    { children, className, disabled, endDecorator, startDecorator, ...props },
+    ref,
+  ) => {
+    return (
+      <RadixMenu.Item disabled={disabled} ref={ref} {...props}>
```

Comment:
As we are spreading the props. Also we can remove it from parameters.
```suggestion
      <RadixMenu.Item  ref={ref} {...props}>
```